src/main/kotlin/org/axonframework/intellij/ide/plugin/markers/publishers/DeadlinePublisherLineMarkerProvider.kt

```diff
@@ -93,7 +94,12 @@ class DeadlinePublisherLineMarkerProvider : LineMarkerProvider {
 
     private fun getDeadlineNameForJava(element: PsiElement): String? {
         val methodCall = element.toUElementOfType<UIdentifier>()
-            ?.getParentOfType(UCallExpression::class.java, true, USimpleNameReferenceExpression::class.java)
+            ?.getParentOfType(
+                UCallExpression::class.java,
+                true,
+                USimpleNameReferenceExpression::class.java,
```

Comment:
Would you mind explaining to me why this is a solution, @Morlack? Just out of curiosity, really.

Reply:
Yes, of course! 

Let's take a look of the UAST tree of this beauty:
```kotlin
deadlineManager.cancelSchedule(PROVISIONER_DELETE_DEADLINE, this.provisioningDeadline);
```

UAST Tree:
```
UQualifiedReferenceExpression
    USimpleNameReferenceExpression (identifier = deadlineManager)
    UCallExpression (kind = UastCallKind(name='method_call'), argCount = 2))
        UIdentifier (Identifier (cancelSchedule))
        USimpleNameReferenceExpression (identifier = PROVISIONER_DELETE_DEADLINE)
        UQualifiedReferenceExpression
            UThisExpression (label = null)
            USimpleNameReferenceExpression (identifier = provisioningDeadline)
```

We want a line marker on the `UIdentifier (Identifier (cancelSchedule))`. So we look up in the UAST tree to see if we can find a UCallExpression that matches a deadline method.
The last varargs argument of the `getParentOfType` contains terminators, if it contains that type of element, it stops searching up the tree. I accounted for the `USimpleNameReferenceExpression` before, so it does not find the call for arguments, just for the Identifier. However, the statement `this.provisioningDeadline` is not a 'simple' reference, but a `UQualifiedReferenceExpression`. So I added that to the terminator list to prevent it from creating multiple line markers for the same method

Reply:
To be even more clear, the following example would not exhibit this issue:
```kotlin
deadlineManager.cancelSchedule(PROVISIONER_DELETE_DEADLINE, "MY_DEADLINE");
```

Since the "MY_DEADLINE" is a ULiteralExpression (it being a String) and PROVISIONER_DELETE_DEADLINE is a USimpleNameReferenceExpression since it's not qualified (having `something.<variable>`)

Reply:
Awesome, thanks for the explanation, @Morlack!